File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean hasDusplicate = false;
        int st;
        for (int i = 0; i < nums.length; i++) {
            st = nums[i];
            for (int j = i +1; j < nums.length; j++) {
                if(st == nums[j]) {
                    hasDusplicate = true;
                    break;
                }
            }
        }
        return  hasDusplicate;
    }
}

File: Data Structures & Algorithms/maximum-product-subarray/submission-0.cpp
class Solution {
   public:
    int maxProduct(vector<int>& nums) {
        int pre = 1, suff = 1;
        int maxi = INT_MIN;
        for (int i = 0; i < nums.size(); i++) {
            if (pre == 0) pre = 1;
            if (suff == 0) suff = 1;
            pre *= nums[i];
            suff *= nums[nums.size() - i - 1];
            maxi = max(maxi, max(pre, suff));
        }
        return maxi;
    }
};
